package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A reputation-based, transaction-oriented platform",
   "main": "index.js",
   "scripts": {
-    "start": "webpack --progress --colors --watch & nodemon server/server.js",
+    "start": "webpack --progress --colors --watch & nodemon server/index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We could change this to `mocha test` so we can easily run `dc run web npm test` to run all the tests